src/librustdoc/html/highlight/sample.rs

```diff
@@ -0,0 +1,8 @@
+#![crate_type = "lib"]
```

Comment:
I think it'd be better to move this into a `test` folder and not in the middle of the source code.